doctr/models/recognition/utils.py

```diff
@@ -59,7 +60,8 @@ def merge_strings(a: str, b: str, overlap_ratio: float) -> str:
         return a_crop + b_crop
 
     # Find best overlap by minimizing Hamming distance + distance from expected overlap size
-    combined_scores = [score + abs(i - expected_overlap) for i, score in enumerate(scores)]
+    expected_overlap_index = expected_overlap - 1
```

Comment:
I think it would be better to adjust directly here (L43):

`expected_overlap = round(len(b) * overlap_ratio) - 2` to -3 ?